source/scripts/Background/index.ts

```diff
@@ -199,13 +199,15 @@ function registerListener() {
 
   browser.runtime.onConnect.addListener((port) => {
     const isPolling = store.getState().vault.isPolling;
+    const isRollux = store.getState().vault.activeNetwork.chainId === 57000;
```

Comment:
Needs to consider rollux mainnet chainId as well (570).
The rpc for testing is this one:

https://rpc.rollux.com
--


<br class="Apple-interchange-newline">